posthog/client.py

```diff
@@ -38,6 +39,7 @@
     CLICKHOUSE_VERIFY,
     TEST,
 )
+from posthog.settings.base_variables import DEBUG
```

Comment:
```suggestion
from django.conf import settings
```

and then `settings.DEBUG`, or `settings.TEST`.

This is the recommended way of accessing django settings & allows easy hooks in tests using django settings overrides.